index.html

```diff
@@ -56,7 +61,7 @@ <h1>My Recent Works</h1>
             <div class="grid works-list">
                 <article class="card work">
                     <div class="background">
-                        <img src="https://picsum.photos/200/300" alt="">
+                        <img src="https://picsum.photos/200/150" alt="">
                     </div>
                     <div class="content">
```

Comment:
- [ ] I'd suggest you populate the work section as it is required to match the original design as much as possible😉 it also improves your page UI(User Interface)👍
- Your Implementation | Figma Design
 ![](https://user-images.githubusercontent.com/44624138/149411136-caa0d940-eafc-44d8-a7bd-c73a0daea50f.png) | ![](https://user-images.githubusercontent.com/44624138/149411097-5798257c-e216-47c0-8cf1-a04d874b16b5.png)